ovos_PHAL_plugin_oauth/__init__.py

```diff
@@ -182,6 +188,47 @@ def handle_oauth_register(self, message):
             # eg. spotify
             self.oauth_skills[skill_id]["needs_creds"] = True
 
+    def handle_refresh_token(self, message):
+        """Refresh oauth token.
+
+        See:
+        https://www.oauth.com/oauth2-servers/making-authenticated-requests/refreshing-an-access-token/
+
+        for details on the procedure.
+        """
+        oauth_id = f"{message.data['skill_id']}_{message.data['app_id']}"
+        # Load all needed data for refresh
+        with OAuthApplicationDatabase() as db:
```

Comment:
After a rebase on `dev`, there's a class variable for this so it doesn't need to be instantiated on every reference